src/scripts/main.js

```diff
@@ -1 +1,59 @@
 'use strict';
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+})
+  .then(data => {
+    document.body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification" class="success">
+      ${data}
+    </div>
+    `);
+  })
+  .catch(err => {
+    document.body.innerHTML += `
+    <div data-qa="notification" class="warning">
+      ${err}
+    </div>
+    `;
+  });
```

Comment:
call it separately
<img width="937" alt="image" src="https://user-images.githubusercontent.com/102411130/200513644-359707ca-a3f4-4e05-ad18-4cad65c32100.png">